share/availability/light/availability_test.go

```diff
@@ -17,6 +16,7 @@ import (
 	core "github.com/tendermint/tendermint/types"
 
 	"github.com/celestiaorg/celestia-app/pkg/da"
+	"github.com/celestiaorg/celestia-app/pkg/namespace"
```

Comment:
It seems like we would need to depend on this type everywhere rather than on low-level namespace at some point. I will likely tackle that during share pkg extraction. It will require some refactorings anyway